openidconnect-signin.js

```diff
@@ -202,13 +206,22 @@ class OpenIDConnectSignin extends LitElement {
 
   _handleClick(e) {
     if (this._signedIn) {
-      this._manager.signoutPopup().catch(error => {
-        // could not log out, at least clear state
-        this._manager.clearStaleState();
-        this._manager.removeUser();
-      });
+      const errorHandler = ()  => {
+          // could not log out, at least clear state
+          this._manager.clearStaleState();
+          this._manager.removeUser();
+      };
+      if(this.useRedirect){
+          this._manager.signoutRedirect().catch(errorHandler);
+      }else{
+          this._manager.signoutPopup().catch(errorHandler);
+      }
     } else {
-      this._manager.signinPopup();
+      if(this.useRedirect){
```

Comment:
Please add spaces before/after the brackets. Also applies to two other lines above.

Reply:
Fixed